Vladyslav_Gurevych/HW6/Task1.py

```diff
@@ -0,0 +1,46 @@
+import os
+
+first_operand = []
+second_operand = []
+operation = []
+line = []
+
+for i in range(1, 201, 2):
+
+    if i < 10:
+        first_operand.append(i)
+    elif 10 <= i < 40:
+        s = int(i / 5)
+        first_operand.append(s)
+    else:
+        d = int(i / 4)
+        first_operand.append(d)
+
+for i in range(100):
+    if i < 10:
+        second_operand.append(i)
+    elif 10 <= i < 40:
+        i = int(i / 4)
+        second_operand.append(i)
+    else:
+        i = int(i / 8)
+        second_operand.append(i)
+
+for i in range(100):
+    if i % 2 == 0:
+        operation.append(1)
+    elif i % 3 == 0:
+        operation.append(2)
+    else:
+        operation.append(3)
+
+for i in range(len(operation)):
+    line.append(f'{first_operand[i]} {second_operand[i]} {operation[i]}')
+
+result = tuple(line)
```

Comment:
it is tuple of strings, but I expect list of tuples

Reply:
![image](https://user-images.githubusercontent.com/25909810/188275488-00ba239b-5649-4845-aedf-b3fdcbc3fe19.png)

Reply:
fixed